### DISCUSSION TITLE
Log error report

### DISCUSSION THREAD

**Discussion by woodrook** [alexdelprete/ha-abb-powerone-pvi-sunspec#116] [Support]
Hi Alex, today I randomly look at the HA Core log and i found many errors like this

"ERROR (SyncWorker_0) [custom_components.abb_powerone_pvi_sunspec.api] (opt_comm_model) Model unknown, report to @alexdelprete on the forum the following data: Manuf.: ABB - Model: -3P37- - Options: 0x0AED/0xFF69/ - OptModel: 0x0AED/0xFF69/ - OptModelInt: 10[0m"

("SyncWorker_*** number always change in each row)

Everything is working well and the integration is perfect

Thanks, Gianluca

**Comment by alexdelprete** [comment]
Yes that simply means your model is not a known one in the internal db. If you tell me the exact model I can add it so in next release you can see the model instead of the strange string in the model sensor. :)

**Comment by woodrook** [comment]
Hi Alex, for sure!!

ABB Solar Inverter
Model: UNO-DM-1.2-TL-PLUS-B-Q

[image: image.png]

__________________________________________________________

Gianluca Rossini

Mail: ***@***.***

Please consider the environment before printing this email



*This e-mail is confidential and may also contain privileged information.
If you are not the intended recipient you are not authorised to read,
print, save, process or disclose this message. If you have received this
message by mistake, please inform the sender immediately and delete this
e-mail, its attachments and any copies. Any use, distribution, reproduction
or disclosure by any person other than the intended recipient is strictly
prohibited and the person responsible may incur penalties. **Thank you!*


Il giorno ven 16 feb 2024 alle ore 13:12 Alessandro Del Prete <
***@***.***> ha scritto:

**Reply by alexdelprete** [reply]
Added. Thanks for reporting it, you'll have to wait for next release to see the model in the sensor.

![image](https://github.com/alexdelprete/ha-abb-powerone-pvi-sunspec/assets/7027842/d77d1d9c-37fd-4384-ba7c-a217afef4767)

**Reply by alexdelprete** [reply]
I pushed a new release version, please let me know if the sensor correctly reports the model. :)

**Comment by woodrook** [comment]
Debug log enabled, looks like integratione find the correct model

Thanks

[image: image.png]

__________________________________________________________

Gianluca Rossini

Mail: ***@***.***

Please consider the environment before printing this email



*This e-mail is confidential and may also contain privileged information.
If you are not the intended recipient you are not authorised to read,
print, save, process or disclose this message. If you have received this
message by mistake, please inform the sender immediately and delete this
e-mail, its attachments and any copies. Any use, distribution, reproduction
or disclosure by any person other than the intended recipient is strictly
prohibited and the person responsible may incur penalties. **Thank you!*


Il giorno lun 19 feb 2024 alle ore 20:42 Alessandro Del Prete <
***@***.***> ha scritto:

**Reply by alexdelprete** [reply]
I don't see the image.

Anyway, no debug log needed, you should see the model on the device page:

![image](https://github.com/alexdelprete/ha-abb-powerone-pvi-sunspec/assets/7027842/c06f6065-02f7-425f-9fbd-9dbdf86cd343)

And also in the Diagnostic section, one of the sensors is the model:

![image](https://github.com/alexdelprete/ha-abb-powerone-pvi-sunspec/assets/7027842/cd52f7e4-55ce-4141-ba10-8ae8404e7d90)

**Comment by woodrook** [comment]
Right!!

[image: image.png]

__________________________________________________________

Gianluca Rossini

Mail: ***@***.***

Please consider the environment before printing this email



*This e-mail is confidential and may also contain privileged information.
If you are not the intended recipient you are not authorised to read,
print, save, process or disclose this message. If you have received this
message by mistake, please inform the sender immediately and delete this
e-mail, its attachments and any copies. Any use, distribution, reproduction
or disclosure by any person other than the intended recipient is strictly
prohibited and the person responsible may incur penalties. **Thank you!*


Il giorno mar 20 feb 2024 alle ore 16:42 Alessandro Del Prete <
***@***.***> ha scritto:
